test/suite/test_oligarch09.py

```diff
@@ -176,6 +176,37 @@ def test_oligarch_read_delete(self):
             else:
                 self.assertEquals(cursor[str(i)], value1)
 
+    def test_oligarch_read_insert(self):
+        uri = "oligarch:test_oligarch08"
+        create_session_config = 'key_format=S,value_format=S'
```

Comment:
For the new tests, could you include the compression config here? (`...,block_compressor={}'.format(self.block_compress)`) Way as well fix the URI to be `oligarch09` like the other functions while you're here.

Reply:
done